src/containers/App.js

```diff
@@ -149,7 +154,10 @@ export default class App extends React.Component {
         }}
         path={path}
         search={doSearch}
-        opened={leftNavIsOpen}/>}
+        opened={leftNavIsOpen}
+        networkNavAnimation={moveWithMenu}
```

Comment:
could you put the network nav inside `<div id='main'>` and then not have to pass it the same animation parameters?